draft-packages/select/KaizenDraft/Select/Select.module.scss

```diff
@@ -98,6 +99,7 @@ $focus-border-color: $color-blue-500;
     border: 6px $border-solid-border-style transparent;
     border-radius: $border-solid-border-radius;
     box-shadow: $shadow-large-box-shadow;
+    z-index: 3;
```

Comment:
This is the menu dropdown. Because it had no previous `z-index` the elements with a `z-index` were "shining" through (you can see it in the first video with the chevrons).